data/core/view.lua

```diff
@@ -121,8 +126,12 @@ end
 
 
 function View:clamp_scroll_position()
-  local max = self:get_scrollable_size() - self.size.y
-  self.scroll.to.y = common.clamp(self.scroll.to.y, 0, max)
+  local scrollsize_y, scrollsize_x = self:get_scrollable_size()
+  scrollsize_x = scrollsize_x or 0 -- FIXME: not every subclass returns the second value
```

Comment:
I think it is fine to leave it as `math.huge`. It'll at least allow users to try to scroll instead of locking them.

Reply:
I left it at `0` to avoid changing the behavior of other classes. Changing it to `math.huge` has amusing effects like scrolling the command palette.